webpack/scenes/RedHatRepositories/components/RepositorySetRepository.js

```diff
@@ -64,12 +66,28 @@ class RepositorySetRepository extends Component {
   }
 
   render() {
-    const { arch, releasever } = this.props;
+    const { arch, releasever, type } = this.props;
+
+    const yStreamHelp = () => (
+      <span translate>
+        This repository has a very specific update policy. Please see additional&nbsp;
```

Comment:
Since I can't contribute much here as a react developer I'll comment on the wording - `very specific` - can this be more ..specific? :roll_eyes:  I don't know what the right word would be - perhaps something like unique, prohibitive, or something that'd convey more meaning

Reply:
Hmm...this wording came from the planning process, as was agreed upon by @bkearney, @rohoover, and @sideangleside. I think you might have to convince them. Here's some of the evolution of the phrase:

> Y-stream repos are not updated once a newer Y-stream is released. so there is literally NO way to handle the 'incremental update to an existing CV workflow' - Rich

> This repo is not maintained and not recommended - Rox

> This repository provides updates at a specific point in the lifecycle
of Red Hat Enterprise Linux. Users of this repository should be very
familiar with the concepts outlined in <link> before using. - BK

> This repository has limited updates. Please see additional <link> documentation <link> prior to use. - Rox

> This repository has a very specific update policy. Please see additional <link> documentation <link> prior to use. -BK

Remember this is a tooltip with limited space. The message strives to be concise while allowing curious users to explore the linked documentation which explains what we mean by a specific update policy.

Reply:
Yep, I'm not asking for a paragraph but I'm looking at 'very specific' and it's making me scratch my head. I have a feeling a user would do the same - I guess that's what the link is for.

Reply:
I think we should leave the explanation for the documentation, how about:

```
This repository is not suggested. Please see additional documentation prior to use.
```

I would have prefered to use `recommended` instead of `suggested` but I don't want to cause confusion with the `recommended repositories` feature.

Reply:
I don't mind changing this (esp. now that two of you have flagged this), but again, we need to consult with @bkearney or @sideangleside (would love some input ;)

I like @sean797's recommendation, as it's simple and to the point, without divulging any confusing information (that's what the document is for, afterall).

Reply:
+1 to @sean797's recommendation. It contains an fairly stern 'dont use this' memo, which is what we want. 

'This repository has a very specific update policy. Please see additional documentation prior to use.' is a correct & factual statement, but it doesn't say much more than 'RTFM'. @sean797's statement says (immediately) what we want them to (not) do. 

If I had my druthers, I would actually not show the Y-stream repositories unless a application setting was set. That is how strongly I feel about deemphasizing the y-stream repositories.